prov/usnic/src/usdf_progress.c

```diff
@@ -124,6 +125,10 @@ usdf_fabric_progression_thread(void *v)
 
 		/* call timer progress each wakeup */
 		usdf_timer_progress(fp);
+
```

Comment:
This domain list access never got a lock.  I think Ben and I discussed putting a lock in the fabric that would be held while accessing this list, though I don't know if you'll start to hit lock ordering issues by introducing it.